docs/tutorials/README.md

```diff
@@ -0,0 +1,401 @@
+
+    # Pick a qubit.
+    qubit = cirq.GridQubit(0, 0)
+
+    # Create a circuit
+    circuit = cirq.Circuit(
+       cirq.X(qubit)**0.5,  # Square root of NOT.
+    )
+    print("Circuit:")
+    print(circuit)
+```
+
+
+You should see the output as 
+
+```console
+
+(0, 0): ───X^0.5───
+
+```
+
+
+### Run the circuit
+
+Now to see what the circuit does when under qsim. Again, copy the following into your VM window:
+
+
+```python
+    simulator = qsimcirq.QSimSimulator()
+    result = simulator.simulate(circuit)
+    print("Result:")
+    print(result)
+```
+
+
+The output will be:
+
+
+```console
+    measurements: (no measurements)
+    output vector: (0.5+0.5j)|0⟩ + (0.5-0.5j)|1⟩
+```
+
+
+You have successfully simulated a quantum circuit on Google Cloud Platform using a Singularity container.
+
+
+### Running your own script
+
+If you want to run a Python script, you can locate a file in the home directory on your VM, then run something like in the container shell
+
+```console
+    $ python3 /homedir/myscript.py
+```
+
+
+### Exit the container
+
+Exit the container by typing cntl-d twice. You will see the output like:
+
+
+```console
+    [root@79804d33f250 /]# exit
+```
+
+
+-----
+
+
+## **Cleaning up**
+
+To avoid incurring charges to your Google Cloud Platform account for the resources used in this tutorial:
+
+
+### Delete the VM
+
+It is usually recommended to delete the entire project you created for this tutorial, but if you want to continue using the project, you can easily either STOP the VM or Delete it. Select the VM, the either select the STOP square, or the DELETE trash can.
+
+
+
+
+![alt_text](../images/image9.png )
+
+
+### Delete the project
+
+The easiest way to eliminate billing is to delete the project you created for the tutorial.
+
+**Caution**: Deleting a project has the following effects:
+
+*   **Everything in the project is deleted.** If you used an existing project for this tutorial, when you delete it, you also delete any other work you've done in the project.
+*   **Custom project IDs are lost.** When you created this project, you might have created a custom project ID that you want to use in the future. To preserve the URLs that use the project ID, such as an **```appspot.com```** URL, delete selected resources inside the project instead of deleting the whole project.
+
+If you plan to explore multiple tutorials and quickstarts, reusing projects can help you avoid exceeding project quota limits.
+
+1. In the Cloud Console, [go to the **Manage Resources** page](https://console.cloud.google.com/iam-admin/projects)
+2. In the project list, select the project that you want to delete and then click **Delete** 
+<img src="../images/image3.png" width="18"/>
+
+
+3. In the dialog, type the project ID and then click **Shut down** to delete the project.
+
+----
+
+
+## **What's next**
+
+Additional tutorial examples are available here:
+
+
+
+*   [Cirq examples](https://cirq.readthedocs.io/en/stable/examples.html)
```

Comment:
I'd remove the links to `examples.html` and the `qsimcirq.ipynb`.  These links are outdated and already in the navigation

Reply:
Typically we have some follow up study offered. Is there a way to add that?